src/xo-app/vm/index.js

```diff
@@ -272,7 +282,22 @@ export default class Vm extends Component {
           </Row>
         </TabPanel>
         <TabPanel>
-          <Debug value={this.state} />
+          <Row>
+            <Col size={6}>
+              {stats && <CpuLineChart data={stats.cpus} endTimestamp={endTimestamp} />}
```

Comment:
I believe xo-server should return the latest timestamp and the interval which should be used to compute the previous ones.